chainermn/optimizers.py

```diff
@@ -62,14 +61,10 @@ def __init__(self, actual_optimizer, communicator):
             'needs_update', False)
         super(_DoubleBufferingOptimizer, self).__setattr__(
             'device', None)
```

Comment:
As I don't think this is used any more, can be removed?

Reply:
Thanks. I will remove it.